studio/components/interfaces/Organization/Documents/DPA.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  ScaffoldSection,
+  ScaffoldSectionContent,
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <ScaffoldSection>
+      <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+        <p className="text-base m-0">Data Processing Addendum (DPA)</p>
+        <div className="space-y-2 text-sm text-foreground-light m-0">
+          <p>
+            For the document to be considered executed, sign and return the signed DPA document to{' '}
```

Comment:
Could reverse order here to get rid of the comma:

Sign and return the DPA document for it to be considered executed.